python/ray/tune/config_parser.py

```diff
@@ -5,133 +5,50 @@
 
 import argparse
 import json
-import numpy as np
-import os
-import random
 
-from ray.tune.trial import Trial, Resources
+from ray.tune.trial import Resources
 
 
-def _resource_json(data):
-    values = json.loads(data)
-    return Resources(values.get('cpu', 0), values.get('gpu', 0))
+def json_to_resources(data):
+    if type(data) is str:
+        data = json.loads(data)
+    return Resources(data.get("cpu", 0), data.get("gpu", 0))
 
 
-def make_parser(description):
+def resources_to_json(resources):
```

Comment:
should make amendments to PR so that #1150 is also supported.

Reply:
Yeah my plan is to rebase either whichever gets merged first.